foursight_core/react/api/react_api.py

```diff
@@ -0,0 +1,447 @@
+            redirect_url = urllib.parse.unquote(redirect_url)
+        return redirect_url
+
+    def react_authorize(self, request: dict, env: str) -> dict:
+        return self.auth.authorize(request, env)
+
+    def react_serve_static_file(self, env: str, **kwargs):
+        return self.react_ui.serve_static_file(env, **kwargs)
+
+    def reactapi_logout(self, request: dict, env: str):
+        domain, context = app.core.get_domain_and_context(request)
+        authtoken_cookie_deletion = create_delete_cookie_string(request=request, name="authtoken", domain=domain)
+        redirect_url = self._get_redirect_url(request, env, domain, context)
+        return self.create_redirect_response(redirect_url, {"set-cookie": authtoken_cookie_deletion})
+
+    def reactapi_header(self, request: dict, env: str):
+        # Note that this route is not protected but/and we return the results from authorize.
+        auth = self.auth.authorize(request, env)
+        data = ReactApi.Cache.header.get(env)
+        if not data:
+            data = self.reactapi_header_nocache(request, env)
+            ReactApi.Cache.header[env] = data
+        data = copy.deepcopy(data)
+        data["auth"] = auth
+        data["timestamp"] = convert_utc_datetime_to_useastern_datetime_string(datetime.datetime.utcnow());
+        response = self.create_standard_response("reactapi_header")
+        response.body = data
+        return response
+
+    def reactapi_header_nocache(self, request: dict, env: str):
+        domain, context = app.core.get_domain_and_context(request)
+        stage_name = app.core.stage.get_stage()
+        default_env = self.envs.get_default_env()
+        aws_credentials = self.auth.get_aws_credentials(env if env else default_env)
+        response = {
+            "app": {
+                "title": app.core.html_main_title,
+                "package": app.core.APP_PACKAGE_NAME,
+                "stage": stage_name,
+                "version": app.core.get_app_version(),
+                "domain": domain,
+                "context": context,
+                "local": app.core.is_running_locally(request),
+                "credentials": {
+                    "aws_account_number": aws_credentials["aws_account_number"],
+                    "aws_account_name": aws_credentials["aws_account_name"]
+                },
+                "launched": app.core.init_load_time,
+                "deployed": app.core.get_lambda_last_modified()
+            },
+            "versions": {
+                "foursight": app.core.get_app_version(),
+                "foursight_core": pkg_resources.get_distribution('foursight-core').version,
+                "dcicutils": pkg_resources.get_distribution('dcicutils').version,
+                "python": platform.python_version(),
+                "chalice": chalice_version
+            },
+        }
+        return response
+
+    def reactapi_info(self, request: dict, env: str):
```

Comment:
I would put in the docstring here for all of these the API path it corresponds to.